bodhi/server/migrations/versions/a96dadf085c2_add_critcal_path_packages.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2019 Red Hat, Inc.
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+add_critcal_path_packages.
+
+Revision ID: a96dadf085c2
+Revises: 8c4d6aad9b78
+Create Date: 2019-02-28 10:14:05.792938
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'a96dadf085c2'
+down_revision = '8c4d6aad9b78'
```

Comment:
This likely needs to be updated, as I merged some commits today with migrations. As of the time of this writing, the head is ```58b7919b942c```.

```suggestion
down_revision = '58b7919b942c'
```